src/nginxconfig/templates/setup.vue

```diff
@@ -163,9 +161,10 @@ THE SOFTWARE.
                 });
 
                 clipboard.on('success', e => {
-                    elm.textContent = 'Copied';
+                    const originalTextCopy = elm.textContent;
+                    elm.textContent = this.$t('templates.app.copied');
                     e.clearSelection();
-                    resetText();
+                    resetText(originalTextCopy);
                 });
 
                 clipboard.on('error', () => {
```

Comment:
This error handler also needs to store the original text so it can reset (I'm not too worried about translating this string for now, feel free to leave it untranslated in the files with a `// TODO: translate` comment)